community/pike/APKBUILD

```diff
@@ -89,96 +85,89 @@ build() {
 
 package() {
 	cd "$builddir"
-	for pkg in $(printf "%s" "$subpackages" | tr ' ' '\n'); do
-		depends="$depends $pkg=$pkgver-r$pkgrel"
-	done
+	depends="gmp nettle nettle-utils zlib readline"
```

Comment:
Please put the `depends` for the package in the `depends` variable at the top of the file. This information is used to control the order in which builds happen from what I understand.